backend/views/admin.pug

```diff
@@ -47,12 +61,12 @@ block content
                             td #{val.publishDate}
                             td
                                 a(href=`/player?base=${encodeURIComponent(val.link)}` target="player") PLAYER
-                                |  | 
+                                |  |
```

Comment:
I believe this same is necessary.
kindly confirm.

Reply:
this is from the base branch. Once it fixed there it would be fine here.